golden-ami/build/bin/network-validator.go

```
@@ -0,0 +1,199 @@
+type endpoint struct {
+	Host        string `yaml:"host"`
+	Ports       []int  `yaml:"ports"`
+	TLSDisabled bool   `yaml:"tlsDisabled"`
+}
+
+func main() {
+	flag.Parse()
+	config := reachabilityConfig{}
+	err := config.LoadFromYaml(*configFilePath)
+	if err != nil {
+		err = fmt.Errorf("Unable to reach config file %v: %v", configFilePath, err)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)